rero_ils/modules/libraries/api.py

```diff
@@ -335,23 +335,30 @@ def get_links_to_me(self, get_pids=False):
         :param get_pids: if True list of linked pids
                          if False count of linked records
         """
+        from ..acq_receipts.api import AcqReceiptsSearch
         from ..patrons.api import PatronsSearch
         links = {}
         ptrn_query = PatronsSearch() \
             .filter('term', libraries__pid=self.pid) \
             .filter('term', roles='librarian')
         loc_query = LocationsSearch() \
             .filter('term', library__pid=self.pid)
+        rpt_query = AcqReceiptsSearch() \
```

Comment:
same here... in contradiction with L344 and L341 XD (do what we say, not what we did XD)